File: HB/pysrc/프로그래머스/레벨0/Day25/연속된_수의_합.py
def solution(num, total):
    answer = []
    if total % num ==0:
        for i in range(- num//2 +1, num//2+1):
            answer.append(total // num+i)
    else:
        for i in range(-num//2+1, num//2+1):
            answer.append(total // num +i)
    return answer

File: SB/src/프로그래머스/레벨0/Day24/연속된_수의_합.java
package 프로그래머스.레벨0.Day24;

public class 연속된_수의_합 {
    class Solution {
        public int[] solution(int num, int total) {
            int[] answer = new int [num];

            int start = (total / num) - ((num - 1) / 2);

            for (int i = 0; i < num; i++) {
                answer[i] = start + i;
            }

            return answer;
        }
    }
}
